test/functional/tools/samples_tool_conf.xml

```diff
@@ -19,6 +19,7 @@
   <tool file="multi_output.xml" />
   <tool file="multi_output_configured.xml" />
   <tool file="multi_output_assign_primary.xml" />
+  <tool file="multi_output_assign_primary_ext_dbkey.xml" />
```

Comment:
Can you add this to the list of tools (``TEST_TOOL_IDS``) tested in test/integration/test_extended_metadata.py.